### PR TITLE
[MLU] add mergedAdam kernel.

### PR DESCRIPTION
**Pull request by ShawnNew**
### PR types

New features

### PR changes

OPs

### Describe

add mlu mergedAdam kernel

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/mlu/test_merged_adam_op_mlu.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```
Comment (ronny1996) [inline python/paddle/fluid/tests/unittests/mlu/test_merged_adam_op_mlu.py]: 2021 -> 2022

**Review by ronny1996 — DISMISSED** [review]
LGTM

**Review by ronny1996 — APPROVED** [review]
LGTM

**Review by qili93 — APPROVED** [review]
LGTM

**Pull request merged**